src/PrestaShopBundle/Resources/views/Admin/Improve/Design/MailTheme/Blocks/generate_mails_form.html.twig

```diff
@@ -0,0 +1,97 @@
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{% block generateMailsFormBlock %}
+<div class="row justify-content-center">
+  <div class="col-xl-10">
+    {{ form_start(generateMailsForm, {'action': path('admin_mail_theme_generate')}) }}
+    <div class="card">
+      <h3 class="card-header">
+        <i class="material-icons">email</i> {{ 'Generate mails'|trans }}
+      </h3>
+
+      <div class="card-block row">
+        <div class="card-text">
+
+          <div class="form-group row">
+            <label class="form-control-label">
+              {{ 'Select your mail theme'|trans }}
+            </label>
+            <div class="col-sm">
+              {{ form_errors(generateMailsForm.mailTheme) }}
+              {{ form_widget(generateMailsForm.mailTheme) }}
+            </div>
+          </div>
+
+          <div class="form-group row">
+            <label class="form-control-label">
+              {{ 'Select your language'|trans }}
+            </label>
+            <div class="col-sm">
+              {{ form_errors(generateMailsForm.language) }}
+              {{ form_widget(generateMailsForm.language) }}
+            </div>
+          </div>
+
+          <div class="form-group row">
+            <label class="form-control-label">
+              {{ ps.label_with_help(
+                ('Select a theme you want to overwrite'|trans),
+              ('Core mail templates are stored in the mails folder but can be overriden by theme proper mails folder. Use this option to overwrite mails from a theme.'|trans({}, 'Admin.MailTheme.Help'))
+              ) }}
+            </label>
+            <div class="col-sm">
+              {{ form_errors(generateMailsForm.theme) }}
+              {{ form_widget(generateMailsForm.theme) }}
+            </div>
+          </div>
+
+          <div class="form-group row">
+            {{ ps.label_with_help(
+              ('Overwrite templates'|trans),
+            ('By default existing mail template files are not modified to avoid erase your modifications, enable this option to force overriding them.'|trans({}, 'Admin.MailTheme.Help'))
```

Comment:
Same here. And I think we can make this wording clearer, what about _By default, existing mail template files are not modifiable to avoid the erase of your modifications. Enable this option to force the override._?

Reply:
Ok, I just replace mail with email, and override with overwrite (it's more accurate)